chapters/data-analysis.tex

```diff
@@ -262,30 +269,53 @@ \subsection{Correcting data entry errors}
 create an automated workflow to identify, correct and document
 occurrences of duplicate entries.
 
-As discussed in the previous chapter,
-looking for duplicated entries is usually part of data quality monitoring,
-and is typically addressed as part of that process.
-So, in practice, you will start writing data cleaning code during data collection.
-The other only other case when changes to the original data points are made during cleaning
-is also directly connected to data quality monitoring:
-it's when you need to correct mistakes in data entry.
-During data quality monitoring, you will inevitably encounter data entry mistakes,
-such as typos and inconsistent values.
-These mistakes should be fixed in the cleaned data set,
-and you should keep a careful record of how they were identified,
-and how the correct value was obtained.
-
 \subsection{Labeling, annotating, and finalizing clean data}
 
+The last step of data cleaning is to label and annotate the data,
+so that all users have the information needed to interact with it.
+There are three key steps: renaming, labeling and recoding. 
+This is a key step to making the data easy to use, but it can be quite repetitive.
+The \texttt{iecodebook} command suite, also part of \texttt{iefieldkit},
+is designed to make some of the most tedious components of this process easier.\sidenote{
+	\url{https://dimewiki.worldbank.org/iecodebook}}
+\index{iecodebook}
+
+First, \textbr{renaming}: for data with an accompanying survey instrument, 
+it is useful to keep the same variable names in the cleaned dataset as in the survey instrument.
+That way it's straightforward to link variables to the relevant survey question.
+Second, \textbf{labeling}: applying labels makes it easier to understand your data as you explore it,
+and thus reduces the risk of small errors making their way through into the analysis stage.
+Variable and value labels should be accurate and concise.\sidenote{
+	\url{https://dimewiki.worldbank.org/Data\_Cleaning\#Applying\_Labels}}
+
+Third, \textbf{recoding}: codes for ``Don't know'', ``Refused to answer'', and
+other non-responses should be recoded into extended missing values.\sidenote{
```

Comment:
"extended missing values" does not exist in all languages. It might even be a Stata terminology. Should we mention Stata here? We should somehow mention that it is important that we do not just delete these values. @luizaandrade , is there equivalent practice in R? 

We could say that the codes must be removed but records of them must still be kept. In Stata that can elegantly be done using extended missing values.

Reply:
As far as I know, there are no extended missings in R. I have looked for it quite a bit. But I like Kris' suggestion.

Reply:
done in 946f782b993ca5188eba496b26f743f952497a90

Reply:
done in 946f782b993ca5188eba496b26f743f952497a90